artifactory/usage/reportusage_test.go

```diff
@@ -35,34 +36,47 @@ func TestReportUsageJson(t *testing.T) {
 	type test struct {
```

Comment:
Let's give it an informative name
```suggestion
	type reportUsageTestCase struct {
```

Reply:
done

Reply:
Let's also move this struct outside of this function to improve readability